ui/hooks/useAlerts.ts

```diff
@@ -37,8 +37,10 @@ const useAlerts = (ownerId: string) => {
       if (!field) {
         return [];
       }
-
-      return alerts.filter((alert) => alert.field === field);
+      const fields = field.split(',');
```

Comment:
Following my other comment about alert keys, this logic shouldn't be needed as every field should have a single alert key, and multiple alerts can reference it.

Reply:
I fixed it.